Frontend/Gmfctn/src/app/core/interceptors/http/auth-http.interseptor.ts

```diff
@@ -1,9 +1,9 @@
 import { HttpErrorResponse, HttpEvent, HttpHandler, HttpInterceptor, HttpRequest } from '@angular/common/http';
 import { Injectable } from '@angular/core';
 import { Router } from '@angular/router';
-import { Observable, of, throwError } from 'rxjs';
+import { EMPTY, Observable, of, throwError } from 'rxjs';
```

Comment:
Unused imports. Please, go around your files and check all of them for unused imports

Reply:
Fixed